app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -157,6 +166,9 @@ class EventsFragment : Fragment(), BottomIconDoubleClick {
         rootView.notification.setOnClickListener {
             findNavController(rootView).navigate(EventsFragmentDirections.actionEventsToNotification())
         }
+        rootView.notificationToolbar.setOnClickListener {
+            findNavController(rootView).navigate(EventsFragmentDirections.actionEventsToNotification())
```

Comment:
Duplication can be removed

Reply:
notification is an ImageView shown when toolbar is hide, notificationToolbar is an icon in the toolbar when toolbar is shown, so it's not duplicate

Reply:
oh, that's not what I mean. I'm talking about `findNavController(rootView).navigate(EventsFragmentDirections.actionEventsToNotification())`. But it is okay. :+1:

Reply:
oh okay, I'm fixing now, besides, build is failing so I need to make some fixes as well